### PR TITLE
chore: correct the system settings

### PR DESCRIPTION
**Pull request by Dousir9**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

- Closes #issue

### CODE REVIEW

**Review by BohuTANG — APPROVED** [review]
👍

File: src/query/settings/src/settings_default.rs [hunk 1]
```diff
@@ -162,7 +162,7 @@ impl DefaultSettings {
                 }),
                 ("max_execute_time", DefaultSettingValue {
                     value: UserSettingValue::UInt64(0),
-                    desc: "Sets the maximum query execution time in seconds. Setting it to 0 means no limit.",
+                    desc: "Sets the maximum query execution time in milliseconds. Setting it to 0 means no limit.",
```
Comment (BohuTANG) [inline src/query/settings/src/settings_default.rs:165]: Oh, the code should make the setting*1000 to milliseconds, let's user configure as seconds is more easy.
The more reasonable setting name is 'max_execute_time_seconds'.

Comment (Dousir9) [reply src/query/settings/src/settings_default.rs:165]: What if user need 500 milliseconds.

Comment (Dousir9) [reply src/query/settings/src/settings_default.rs:165]: However, 500 milliseconds is very small, and users generally do not need this.

**Review by sundy-li — APPROVED** [review]

**Pull request merged**